### ISSUE TITLE
Once the server is down, there is not any error message when you try to share with ownCloud

### ISSUE THREAD

**Issue by Dianafg76** [owncloud/client#4484]
### Expected behaviour
An error should appear letting the user know that there is not connection 

### Actual behaviour
After clicking on "share with ownCloud" nothing happens. Meaning nothing that the sharing view is not open neither any error message for the user to know what's going on.

LOG
```
02-18 09:14:25:215 0x7fb959d1b070 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "SHARE:NOTCONNECTED:/Users/Diana/Https_federatedsharing/shareadmin"
02-18 09:14:40:298 0x7fb959d1b070 OCC::ConnectionValidator::checkServerAndAuth: Checking server and authentication
02-18 09:14:40:299 0x7fb959d1b070 OCC::ConnectionValidator::checkServerAndAuth: Trying to look up system proxy
02-18 09:14:40:299 0x7fb95c425690 OCC::SystemProxyRunnable::run: virtual void OCC::SystemProxyRunnable::run() Starting system proxy lookup
02-18 09:14:40:303 0x7fb959d1b070 OCC::ConnectionValidator::systemProxyLookupDone: No system proxy set by OS
02-18 09:14:40:304 0x7fb959d1b070 OCC::AbstractNetworkJob::start: !!! OCC::CheckServerJob created for "https://xxxxxx” + "status.php" "OCC::ConnectionValidator"
02-18 09:14:40:665 0x7fb959d1b070 OCC::AbstractNetworkJob::slotFinished: void OCC::AbstractNetworkJob::slotFinished() 1 "Connection refused" QVariant(Invalid)
02-18 09:14:40:665 0x7fb959d1b070 OCC::ConnectionValidator::slotNoStatusFound: void OCC::ConnectionValidator::slotNoStatusFound(QNetworkReply *) 1 "Connection refused"
02-18 09:14:40:666 0x7fb959d1b070 OCC::AccountSettings::showConnectionLabel: "No connection to ownCloud at <a href="https://docker.oc.solidgear.es:55155">https://xxxxxxxxx</a>.
Connection refused"
02-18 09:14:40:668 0x7fb959d1b070 OCC::FolderMan::slotAccountStateChanged: Account "admin@xxxxxxxxxxxxx” disconnected, terminating or descheduling sync folders
02-18 09:14:40:668 0x7fb959d1b070 OCC::CheckServerJob::finished: error: status.php replied  0 ""
```

### Steps to reproduce
1. You have user1(server1), and user2(server2)
2. Go to the Desktop client with user1(server1)
3. Create a new folder (e.g Hello)
4. Turn down server1 (maintenance mode on)

![screen shot 2016-02-18 at 09 25 04](https://cloud.githubusercontent.com/assets/12032058/13137483/9d0136d8-d621-11e5-8742-274f248a5cb0.png)

5. Access to user1(server1) synced folder
6. Select to share with ownCloud the folder "Hello"

### Server configuration
Desktop v 2.1.1
Server v {"installed":true,"maintenance":false,"version":"9.0.0.11","versionstring":"9.0 beta 1","edition":""}

**Comment by rullzer** [comment]
PR with possible fix in #4493

**Comment by Dianafg76** [comment]
I can reproduce this error
